Apps/WebClient/src/appsettings.Development.json

```diff
@@ -67,6 +67,7 @@
   ],
   "ContentSecurityPolicy": {
     "connect-src": "https://spt.apps.gov.bc.ca/com.snowplowanalytics.snowplow/tp2 https://sso-dev.pathfinder.gov.bc.ca/ http://localhost:*",
-    "frame-src": "https://sso-dev.pathfinder.gov.bc.ca/"
+    "frame-src": "https://sso-dev.pathfinder.gov.bc.ca/",
+    "script-src": "https://sp-js.apps.gov.bc.ca/MDWay3UqFnIiGVLIo7aoMi4xMC4y.js"

```

Comment:
Do you really need the file name in it? i thought just the domain would do it!

Reply:
I dont think is necessary to name the file, you are right the domain would work as well. However I believe having the file in there makes sure that even if the domain gets compromised only that particular file will be allowed from that external source